trunk/templates/wc-yotpo-settings.php

```diff
@@ -135,7 +138,12 @@ function wc_display_yotpo_settings($success_type = false) {
                      </tr>
                      <tr valign='top' id='yotpo-sync-widget-ids-row' style='display:none'>
                        <th scope='row'><div>Synchronise the widgets' codes:</div></th>
+                       <!-- <td><button type='button' id='yotpo-sync-widget-ids' class='button-secondary'>Sync</button></td> -->
                        <td><input type='submit' name='yotpo_sync_ids' value='Sync' class='button-primary' id='yotpo_sync_ids'/></td>
+                     </tr>
+                     <tr valign='top'>
+                       <th scope='row'><div>Reset the widgets' codes:</div></th>
+                       <td><input type='submit' name='yotpo_remove_ids' value='Reset' class='button-secondary' id='yotpo_remove_ids'/></td>
```

Comment:
Why should they remove it? Checking out the checkout should revert them to V2, shouldn't it?